app/controllers/api/logs_controller.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class Api::LogsController < ApiController
+  self.permitted_attrs = :encryptable_id
+
+  def index(options = {})
+    authorize(team, :index?, policy_class: LogPolicy)
+    render({ json: fetch_entries,
+             each_serializer: list_serializer,
+             root: 'Logs_'.pluralize }
+      .merge(render_options)
+      .merge(options.fetch(:render_options, {})))
+  end
+
+  def fetch_entries
+    PaperTrail.serializer = JSON
+    logs = current_user.encryptables.find_by!(id: params[:encryptable_id]).versions
+    if params[:load]
+      logs.last(params[:load]).sort { |a, b| b.created_at <=> a.created_at }
+    else
+      logs.last(10).sort { |a, b| b.created_at <=> a.created_at }
+    end
+  end
+
+  def list_serializer
```

Comment:
do not override this method here.
just define used serializer directly on line 8
do not use constantize ... just declare class method directly